javascript-data-model/order-history.js

```diff
@@ -0,0 +1,78 @@
+var orderHistory = [
+  {
+    orderPlaced: 'August 4,2020',
+    total: '$34',
+    shipTo: 'JS Masher',
+    orderNum: '19837409184738',
+    delivered: 'August 8, 2020',
+    items: [
+      {
+        productType: 'book',
+        img: 'src',
+        title: 'random title',
+        author: 'random auth',
```

Comment:
You've got plenty of time left on this exercise, please put in information that better approximates the items you see. Please do this for the rest of your items as well.